Resize talent icons, remove apostrophes

MGatner opened the pull request:
Fix for https://github.com/heroespatchnotes/heroes-talents/issues/21 and https://github.com/heroespatchnotes/heroes-talents/issues/25

MGatner commented:
@stuaroo I went with the following for ability/talent icons:
magick mogrify -resize 64x64 -strip -depth 8 *.png
It is the smallest file size I could get without big quality difference. Take a look at them and see if they look alright to you. I'm glad to tweak the command, but now that we're including the PNGs straight from HeroesDataParser in raw/ I don't feel bad about offering a smaller stripped version.

stuaroo commented:
Quality looks good to me on the resized icons.

I noticed Fenix and Lucio now have sub abilities but they're getting added under a `"0"` key, so I think we'll want to add a more meaningful name to these before merging.  

Also, in the category of humorous but too hard to fix, I noticed that Fenix's default W uses `storm_ui_icon_fenix_w_2` icon (which we label W1) while the sub icon is `storm_ui_icon_fenix_w_1` (which we have as W2)!

MGatner commented:
Bummer on the Fenix icon names! I didn’t notice that, but yeah I think fixing that for consistency would be unnecessarily interventionist. 

Good catch on the subunit abilities. I’ll look into what is getting them a 0. Any thoughts on what they actually should be? They are good to have their for their description text, but they aren’t really “abilities”.

stuaroo commented:
Yeah, both the Fenix and Lucio subunit abilities are really different states for the same ability, but they are good to have for their description text like you say.  I'm not really sure the best way to handle these; the easy way would be to add them as another basic ability using the same hotkey (W), but that's not a great representation of how they actually work.  Thinking about it more though, maybe it is, it's kind of like Stitches's Q/1 where they share a cooldown and only one can be cast at a time.  Could also leave them under a named subunit, I'm not sure, what were you thinking?

MGatner commented:
@stuaroo check out the updates. @koliva8245 updated HeroesDataParser to include abilityLinkId on talents so we don't have to figure them out ourselves! There are some differences though (see https://github.com/koliva8245/HeroesDataParser/issues/24).

Other changes:
* added isQuest for quest talents
* added manaPerSecond as discussed
* fixed subunits for Fenix & Lucio

Take a look and do some QC when you have a chance. I expect a patch from Blizzard probably on Tuesday so I'd like to have as much of this stage of the process resolved by then as possible.

stuaroo commented:
Besides the differences noted in your HeroesDataParser issue, there were just a few weird handling issues for Lucio and Fenix.  Lucio's Crossfade talents all switched from W1 to W2, which still works since both ability states apply in those cases, so this is more of an FYI and probably doesn't need to be looked at.   

A lot of Fenix's Weapon Mode talents switched and now some are incorrect.  Currently W1 is Repeater Cannon and W2 is Phase Bomb, and here are the talents that point to the other ability state:

* Target Acquired changed from W1 to W2 (this is a Repeater Cannon-only talent)
* Inhibiting Energy changed from W2 to W1 (this is a Phase Bomb-only talent)
* Offensive Cadence changed from W1 to W2 (this is a Repeater Cannon-only talent)

A few other talents also changed (Arsenal Synergy, Mobile Offense, Singularity Charge) but as their descriptions reference both Weapon Modes I'm not as concerned about those.

I'd be a little surprised to see a Tuesday patch as usually Tuesday's are reserved for larger patches that are already on the PTR.  A Wednesday balance patch does seem likely though.

MGatner commented:
@stuaroo for a normal game data update after patch I'd feel empowered to do the merge but since this has so many things on the backlog I'd still like your input. @koliva8245 has a fix coming for the Fenix & Lucio weirdness so not sure if you'd prefer to wait on that and/or sort through the updates fro this patch?

stuaroo commented:
I say go ahead with the merge.  Like you mentioned, the only things that are still a little off are Fenix/Lucio and I'm fine with waiting a bit for a HeroesDataParser update to clear those up.  And if someone really needs those ability links updated in the meantime it will be an easy manual update.
